Lib/test/test_importlib/extension/test_loader.py

```diff
@@ -29,6 +29,25 @@ def test_load_module_API(self):
         with self.assertRaises(ImportError):
             self.load_module('XXX')
 
+    def test_type_error(self):
+        with self.assertRaises(TypeError) as cm:
+            other = self.machinery.ExtensionFileLoader(
+                fullname=util.EXTENSIONS.name,
+                name=util.EXTENSIONS.name,
+                path=util.EXTENSIONS.file_path)
+
+        assert "name" in str(cm.exception)
```

Comment:
Please use the `unittest` methods for testing.
```suggestion
        self.assertIn("name", str(cm.exception))
```